documentation/docs/Exercises/Exercise 4.md

```diff
@@ -91,7 +97,7 @@ You can find `Cosmoshub channel id` under [Testnet Release](/testnet-releases.md
 [cosmos-key-name] is the one you generated in setup 5 above.
 
 ```bash
-gaiad tx ibc-transfer transfer transfer [Cosmoshub channel id] [axelar address] [amount]uphoton --from [cosmos-key-name] -y -b block
+gaiad tx ibc-transfer transfer transfer [Cosmoshub channel id] [axelar address] packet-timeout-timestamp 0 [amount]uphoton --from [cosmos-key-name] -y -b block
```

Comment:
@mindstyle85 `packet-timeout-timestamp 0` should be passed as a flag, it should be `--packet-timeout-timestamp 0`